modules/twinkleblock.js

```diff
@@ -1914,7 +1914,7 @@ Twinkle.block.callback.main = function twinkleblockcallbackMain(pageobj) {
 	params.indefinite = Morebits.string.isInfinity(params.expiry);
 
 	if (Twinkle.getPref('blankTalkpageOnIndefBlock') && params.template !== 'uw-lblock' && params.indefinite) {
-		Morebits.status.info('Info', 'Blanking talk page per preferences and creating a new level 2 heading for the date');
+		Morebits.status.info('Info', 'Blanking talk page per preferences and creating a new talk page section for this month');
```

Comment:
@molly I didn't even think about this one!  There's another in line [https://github.com/wikimedia-gadgets/twinkle/pull/1294/files#diff-eddb511f8eaac257a180d1631b66386089bd898ed7b3725c0577f9732882700cR1936 1936], wanna grab that?

Reply:
Oh, good catch! I've made the change (and squashed the commits again).